scripts/convert_to_schemapack.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+"""Script to convert linkML schema to schemapack definition"""
+
+from pathlib import Path
+
+import yaml
+from schemapack._internals.dump import dump_schemapack
+from schemapack.spec.schemapack import (
+    ClassDefinition,
+    ContentSchema,
+    IDSpec,
+    MandatoryRelationSpec,
+    MultipleRelationSpec,
+    Relation,
+    SchemaPack,
+)
+from script_utils.cli import run
+
+HERE = Path(__file__).parent.resolve()
+SCHEMA_FOLDER = HERE.parent / "src" / "schema" / "submission.schemapack.yaml"
+LINKML_SCHEMA = HERE.parent / "src" / "schema" / "submission.yaml"
+RELATIONS_CONFIG = HERE.parent / "relations_config.yaml"
+CLASS_CONTENT_FOLDER = HERE.parent / "src" / "content_schema_artifacts"
+EXCLUDED_CLASSES = HERE.parent / "exclude_config.yaml"
+
+
+def load_yaml(path: Path):
+    """Loads the data from a yaml file"""
+    with path.open("r", encoding="utf-8") as file:
+        return yaml.safe_load(file)
+
+
+def construct_mandatory(relation_name: str, class_schema: dict):
+    """Returns the constraints in the relationship modality"""
+    return MandatoryRelationSpec(
+        origin=False,
+        target=class_schema.get("slot_usage", {})
+        .get(relation_name, {})
+        .get("required", False),
+    )
+
+
+def construct_multiple():
+    """Returns the schemapack specification for 1 to n relationship"""
+    return MultipleRelationSpec(origin=True, target=False)
+
+
+def construct_relation(
+    class_relations: dict, class_schema: dict
+) -> dict[str, Relation]:
+    """Creates objects of Relations from the relation list of a class"""
+
+    return {
+        key: Relation(
+            description=None,
+            targetClass=value.get("targetClass"),
+            mandatory=construct_mandatory(key, class_schema),
+            multiple=construct_multiple(),
+        )
+        for key, value in class_relations.items()
+    }
+
+
+def construct_content_schema(path: Path) -> str:
+    """Reads the content schema"""
+    with path.open("r", encoding="utf-8") as file:
+        content = file.read()
+    return content
+
+
+def construct_schemapack_class(
+    class_name: str, class_relations: dict, class_schema: dict
+) -> ClassDefinition:
+    """Creates ClassDefinition object"""
+    return ClassDefinition(
+        description=class_schema["description"],
+        id=IDSpec(propertyName="alias", description=None),
+        content=ContentSchema(
+            json_schema=construct_content_schema(
+                Path(f"{CLASS_CONTENT_FOLDER}/{class_name}.json")
+            )
+        ),
```

Comment:
The function `construct_content_schema` is a re-implementation of [Path.read_text()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text). However, you don't need to do the schema loading at all, since it's done automatically by [a `ClassDefinition` validator](https://github.com/ghga-de/schemapack/blob/f476198925f59877d5b2dbdb8039fed1b64f16f8/src/schemapack/_internals/spec/schemapack.py#L214-L278):
```suggestion
        content=Path(f"{CLASS_CONTENT_FOLDER}/{class_name}.json"),
```

Reply:
I removed the `construct_content_schema` function. 

Path is an incompatible type of `ClassDefinition` content. I am not sure if it is better the suppress the typing error or create an instance of `ContentSchema` with `Path.read_text()`.